### PR TITLE
Added some dependencies for VS 19 image

### PR DESCRIPTION
**Pull request by melihmutlu**
Issue #2 
Added icu, lz4, zstd

### CODE REVIEW

File: docker/windows_ci_vs_2019 [hunk 1]
```diff
@@ -4,12 +4,14 @@ FROM windows_ci_vs_base
 
 COPY `
   scripts/windows_install_vs_2019.ps1 `
+  scripts/windows_install_deps.ps1 `
```
Comment (anarazel) [inline docker/windows_ci_vs_2019:7]: I think this should be in the windows_ci_vs_base layer. That way we can test both vs_2019 and vs_2021 eventually, with the tools available.

File: docker/windows_ci_vs_2019 [hunk 2]
```diff
@@ -4,12 +4,14 @@ FROM windows_ci_vs_base
...
   scripts/
 
 RUN `
   `
   scripts\windows_install_vs_2019.ps1 ; `
+  scripts\windows_install_deps.ps1 ; `
   <# Adding path to vcvarsall.bat to PATH so user of container don't need to  know the full path #> `
-  [Environment]::SetEnvironmentVariable('PATH',  'C:\BuildTools\VC\Auxiliary\Build;' + [Environment]::GetEnvironmentVariable('PATH', 'Machine'), 'Machine') ; `
+  [Environment]::SetEnvironmentVariable('PATH',  'C:\BuildTools\VC\Auxiliary\Build;C:\icu;C:\lz4;C:\zstd;' + [Environment]::GetEnvironmentVariable('PATH', 'Machine'), 'Machine') ; `
```
[the remark this answers is not in the document]
Comment (melihmutlu) [reply docker/windows_ci_vs_2019]: It now adds each library to the Path in a seperate line. Is this okay or would it be better to add just pg dependency paths in a new single line?

File: scripts/windows_install_deps.ps1 [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    if (!$?) { throw 'cmdfail' }
+    Remove-Item "c:\$($DependencyName).zip" -Force
+}
+
+
+DownloadDependency "icu" "https://github.com/unicode-org/icu/releases/download/release-70-1/icu4c-70_1-Win64-MSVC2019.zip";
+InstallDependency "icu";
+
+DownloadDependency "lz4" "https://github.com/lz4/lz4/releases/download/v1.9.3/lz4_win64_v1_9_3.zip";
+InstallDependency "lz4";
```
[the remark this answers is not in the document]
Comment (melihmutlu) [reply scripts/windows_install_deps.ps1]: That would be nicer. But I'm not sure how to handle download urls for specified versions of each library. Any suggestion?

File: docker/windows_ci_vs_2019 (hunk 2 above)
[the remark this answers is not in the document]
Comment (anarazel) [reply docker/windows_ci_vs_2019]: Think that's good.

File: scripts/windows_install_pg_deps.ps1 [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    curl.exe -sSL -o "c:\$($DependencyName).zip" $SourceUri;
+    if (!$?) { throw 'cmdfail' }
+
+    echo "installing $($DependencyName)"
+    7z.exe x "c:\$($DependencyName).zip" -o"c:\$($DependencyName)"
+    if (!$?) { throw 'cmdfail' }
+    Remove-Item "c:\$($DependencyName).zip" -Force
+}
+
+DownloadAndInstallDependency "icu" "https://github.com/unicode-org/icu/releases/download/release-70-1/icu4c-70_1-Win64-MSVC2019.zip";
```
Comment (anarazel) [inline scripts/windows_install_pg_deps.ps1:15]: Looks like there's 71.1 now

**Review by anarazel — APPROVED** [review]
Please merge after updating ICU version.

File: scripts/windows_install_pg_deps.ps1 (hunk 4 above)
Comment (melihmutlu) [reply scripts/windows_install_pg_deps.ps1]: Updated icu

**Pull request merged**